setup/openchange_neworganization

```diff
@@ -62,6 +63,10 @@ def setup_path(*args):
 provisionnames = openchange.guess_names_from_smbconf(
     lp, creds, opts.firstorg, opts.firstou)
 
-if openchange.provision_organization(setup_path, provisionnames, lp, creds):
-    openchange.register(setup_path, provisionnames, lp, creds)
-    openchange.openchangedb_new_organization(provisionnames, lp)
\ No newline at end of file
+if opts.deprovision:
+    openchange.deprovision_organization(setup_path, provisionnames, lp, creds)
+    openchange.unregister(setup_path, provisionnames, lp, creds)
+else:
```

Comment:
I consider this clearer, future changes on provision code may not depend on provision_organization result